patterns/minimising-data-size-transmitted.md

```diff
@@ -0,0 +1,38 @@
+# Minimising the size of data transmitted
+
+## Version
+1.0
+
+## Submitted By
+Sarah Hsu
+
+## Published Date
+TBD
+
+## Intent
+Minimising the size of data transmitted
+
+## Tags
+Cloud, Networking, Cloud Engineer, Small
+
+## Problem
+From an energy-efficiency perspective, it's better to minimise the size of the data transmitted so that less energy is required because the network traffic is reduced. 
+
+## Solution
+Minimise the size of data transmitted by compressing files or payloads.
+
+## SCI Impact
+`SCI = (E * I) + M per R`
```

Comment:
There is an additional/increased `E` component and potentially `I` with requiring eletricity/compute to compress and decompress the data. Should this be called out in the pattern?

Reply:
I have attempted to address this comment. Let's have a quick discussion during the call later?